<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GdalToolsWidget</class>
 <widget class="QWidget" name="GdalToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>163</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Contour</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Input file (raster)</string>
       </property>
       <property name="buddy">
        <cstring>inSelector</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Output file for contour lines (vector)</string>
       </property>
       <property name="buddy">
        <cstring>outSelector</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>I&amp;nterval between contour lines</string>
       </property>
       <property name="buddy">
        <cstring>intervalDSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="intervalDSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>2140000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="attributeCheck">
         <property name="text">
          <string>&amp;Attribute name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>If not provided, no elevation attribute is attached.</string>
         </property>
         <property name="indent">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="attributeEdit">
         <property name="text">
          <string>ELEV</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="GdalToolsInOutSelector" name="outSelector" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="GdalToolsInOutSelector" name="inSelector" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GdalToolsInOutSelector</class>
   <extends>QWidget</extends>
   <header>.inOutSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
